backend/src/data/migrations/20210822152011_add_comment_reactions_table.ts

```diff
@@ -0,0 +1,20 @@
+import { Knex } from 'knex';
+import { getFormattedISODate } from '~/helpers/helpers';
+
+const TABLE_NAME = 'comment_reactions';
+
+export async function up(knex: Knex): Promise<void> {
+  const dateNowISO = getFormattedISODate(new Date());
+
+  return knex.schema.createTable(TABLE_NAME, (table) => {
+    table.increments('id').primary();
+    table.integer('user_id').notNullable().references('id').inTable('users').onDelete('CASCADE');
+    table.integer('comment_id').notNullable().references('id').inTable('comments').onDelete('CASCADE');
```

Comment:
```suggestion
    table.integer('user_id').notNullable().references('id').inTable('users');
    table.integer('comment_id').notNullable().references('id').inTable('comments');
```
avoid CASCADE in out project